tests/clients/test_base.py

```diff
@@ -35,9 +35,9 @@
     _ErrorSchema,
 )
 
-MOCK_SERVICE_NAME = "test-service"
+MOCK_SERVICE_URL = "https://test-service.example.com/"
 MOCK_ENDPOINT = "/endpoint"
-MOCK_SERVICE_URL = "https://test-service.example.com/endpoint"
+MOCK_ENDPOINT_URL = "https://test-service.example.com/endpoint"
```

Comment:
**Nit**: Since this value derived from the previous 2 lines, would it make sense to actually derive it here along the lines how the url is calculated inside the BaseClient? I am a bit torn between readability of code (as it is), and clarity of intent (defining this line in terms of the previous two), something like:

```suggestion
MOCK_ENDPOINT_URL = MOCK_SERVICE_URL.rstrip("/") + "/" + MOCK_ENDPOINT.lstrip("/")
```

Reply:
Good suggestion, though I think in this case I think I prefer the fixed strings. My reasoning is that your suggestion would re-implement the logic that's under test, which I think is best avoided when possible. In this case, we validate that a particular input results in a particular output, which feels more in the spirit of unit tests.